test/api/test_history_contents.py

```diff
@@ -1,11 +1,10 @@
-from base import api
 import json
 
-from .helpers import TestsDatasets
-from .helpers import LibraryPopulator
-from .helpers import DatasetCollectionPopulator
-from requests import delete
-from requests import put
+from requests import delete, put
+
+from base import api
+
+from .helpers import DatasetCollectionPopulator, LibraryPopulator, TestsDatasets
```

Comment:
This is sort of a huge change and I don't really know the best answer. Maybe we should start migrating the tests to ``lib/galaxy_test`` or ``lib/galaxy/test``. This is sort of recommended by this guide http://python-packaging.readthedocs.io/en/latest/testing.html if you want to distribute your tests with your Python package (and I think we will want to do that at some point). Usually I like to prototype this stuff out with a different project - but none of the other projects has this problem per se.

Ping @natefoo - any thoughts?